docs/src/man/reading.md

```diff
@@ -164,16 +168,17 @@ tcagTTAAGATGGGAT
 
 To read a file containing such records, one could use:
 ```julia
-reader = open(FASTQReader, "reads.fastq", Seq.SANGER_QUAL_ENCODING)
-seqrec = eltype(reader)()
-while !eof(reader)
-    read!(reader, seqrec)
-    # ... process sequence
+reader = open(FASTQReader, "reads.fastq")
```

Comment:
Okay, but safety first is a Julian way so I think the copy iteration should be placed first. I added an in-place reading syntax with a comment after that.